packages/foreman/foreman/foreman.spec

```diff
@@ -216,8 +214,6 @@ BuildRequires: %{?scl_prefix}rubygem(sprockets) >= 4.0
 BuildRequires: %{?scl_prefix}rubygem(sprockets) < 5.0
 BuildRequires: %{?scl_prefix}rubygem(sprockets-rails) >= 3.0
 BuildRequires: %{?scl_prefix}rubygem(sprockets-rails) < 4.0
-BuildRequires: %{?scl_prefix}rubygem(record_tag_helper) >= 1.0
-BuildRequires: %{?scl_prefix}rubygem(record_tag_helper) < 2.0
```

Comment:
We shouldn't forget to drop this, preferably before branching. @zjhuntin @pcreech @Odilhao.

Reply:
?

Reply:
The `rubygem-record_tag_helper` is now no longer used so we can retire the package.

Reply:
ah! this = the package, not this = this hunk. gotcha.

Reply:
Yes, early morning reviews aren't the best. I opened https://github.com/theforeman/foreman-packaging/pull/7513 which should do this.